src/components/marginals/Navbar/DesktopNav.jsx

```diff
@@ -60,7 +61,7 @@ function DesktopNav() {
             </ul>
           </NavCenter>
           <NavRight>
-            <AuthButton outline />
+            <AuthButton />
```

Comment:
Delete the authButton and use https://github.com/dscnitrourkela/project-raisin/blob/main/src/components/shared/Button.jsx with an outline prop.